db/migrate/20220226092154_add_recording_status_callback_url_and_recording_status_callback_method_to_phone_calls.rb

```diff
@@ -0,0 +1,8 @@
+class AddRecordingStatusCallbackUrlAndRecordingStatusCallbackMethodToPhoneCalls < ActiveRecord::Migration[7.0]
+  def change
+    change_table :phone_calls do |t|
```

Comment:
Why not just put this in the recordings table?

Reply:
We need to save it in phone_calls table because for the outbound call they put those fields when creating a phone call, so we need to save it first.

`recordingStatusCallback` attribute of `<Record>` verb is optional, so when user doesn't specify it, it will copy the value from phone calls table recordings table. `<Record /><Record recordingStatusCallback="https://another-domain.com" />`